website/src/components/lifeCycle/styles.module.css

```diff
@@ -0,0 +1,13 @@
+.lifecycle {
+    background-color: #047377; /* Teal background to align with dbt Labs' color */
+    color: #fff; /* Light text color for contrast */
+    font-size: 0.4em; /* Slightly smaller text */
```

Comment:
Regarding the `font-size` - `em` is making the font size relative to the parent element the lifecyle component is a part of, so in this case the header size. One situation where this might cause an issue is if the lifecycle component is used outside of the header.

In this case, `.4em` would make the font size relative to it's parent. For example, if i add the lifecycle component on it's own line, it's parent element becomes a `div`, which has a smaller font size set than headers. `.4em` of the div's font size (`1.125rem`) would end up small and difficult to read.

![image](https://github.com/dbt-labs/docs.getdbt.com/assets/13647110/85ae18cb-0b29-491e-a624-5955490565da)

One adjustment you can make here is switch from `em` to `rem`, which makes the font size relative to the root, which is usually `1rem`. If you adjust the `font-size` to use `rem`, that size will remain consistent no matter where the component is used. `.4rem` would be pretty small, but bumping it up a bit should get you close to the font size you had when using `em`. 

Using `.9rem` for example would say 'this text should always be slightly smaller than the default font size for the site.'